src/feed/feed.service.ts

```
@@ -9,6 +9,7 @@ import { UpdateFeedDto } from './dto/update-feed.dto';
 import { Job } from "../jobs/entities/job.entity"
 import { NotificationsService } from 'src/notifications/notifications.service';
 import { JobStatus } from './enums/job-status.enum';
+import { Report } from './entities/report.entity';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Report'. Consider to delete it or rename it.

'Report' is defined here:


(lint/suspicious/noRedeclare)